src/index.html

```diff
@@ -2,13 +2,54 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item ">
+            <a href="index.html" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
```

Comment:
It's better to use anchor links
```suggestion
            <a href="#apple" class="nav__link is-active">APPLE</a>
          </li>
          <li class="nav__item">
            <a href="#samsung" class="nav__link">SAMSUNG</a>
          </li>
          <li class="nav__item">
            <a href="#phones" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
```